src/main/paradox/docs/getting-started/running-nexus/index.md

```diff
@@ -47,6 +47,18 @@ On macOS and Windows, Docker effectively runs containers inside a VM created by
 Nexus requires at least **2 CPUs** and **8 GiB** of memory in total. You can increase the limits
 in Docker settings in the menu *Preferences* > *Advanced*.
 
+For a proper evaluation using Docker Swarm or Minikube/Kubernetes, at leat **16GiB** is needed to run the
+provided templates. Feel free to tweak memory limits in order to fit your hardware constraints. At the cost
+of a slower startup and a decreased overall performance, you should be able to go as low as:
+
+|    Service    | Memory [MiB] |
+|:-------------:|:------------:|
+| Cassandra     |          512 |
```

Comment:
do cassandra and es run with such low values?

Reply:
Yes they do